include/lsst/afw/table/io/OutputArchive.h

```diff
@@ -137,7 +138,24 @@ class OutputArchiveHandle {
     /**
      *  Save a nested Persistable to the same archive.
      *
-     *  @copydoc OutputArchive::put.
+     *  If permissive is true and obj->isPersistable() is false, the object will not
+     *  be saved but 0 will be returned instead of throwing an exception.
```

Comment:
All I know is that Doxygen was warning that there was no `OutputArchive::put` method to copy documentation from.  Doxygen is still emitting the same warning for several other afw objects if you'd like to debug further; I just thought I'd fix these ones since I was touching the file anyway.

Reply:
I suspect the problem might be that `OutputArchive::put` is an ambiguous reference; try giving a specific argument list.